hq/app/schedule/IamMessages.scala

```diff
@@ -10,6 +10,15 @@ object IamMessages {
 
   def warningSubject(account: AwsAccount): String = s"Action ${account.name}: Insecure credentials"
   def finalSubject(account: AwsAccount): String = s"Action ${account.name}: Deactivating credentials tomorrow"
+  def disabledUsersSubject(account: AwsAccount): String = s"AWS IAM User(s) DISABLED in ${account.name} Account"
+
+  def disabledUsersMessage(users: Seq[VulnerableUser]): String = {
+    s"""
+       |The following Permanent IAM user(s) have been disabled today: ${users.map(_.username).mkString(",")}.
+       |Please check Security HQ (https://security-hq.gutools.co.uk/iam) to remediate this.
```

Comment:
(Optional) How about "Please check Security HQ to review the IAM users in your account" instead? As we're not necessarily expecting anything to be remediated are we?

I guess we could also include instructions like: "If you still require the disabled user, you should add new access credentials and ensure their access keys are regularly rotated going forwards, or MFA is added for human users. If you no longer require the disabled user, they should be deleted."

Reply:
Great suggestion, thank you!